extensions/registration-service/src/test/java/org/eclipse/dataspaceconnector/registration/api/RegistrationServiceTest.java

```diff
@@ -45,7 +46,7 @@ class RegistrationServiceTest {
     Monitor monitor = mock(Monitor.class);
     ParticipantStore participantStore = mock(ParticipantStore.class);
     DtoTransformerRegistry dtoTransformerRegistry = mock(DtoTransformerRegistry.class);
-    RegistrationService service = new RegistrationService(monitor, participantStore, dtoTransformerRegistry);
+    RegistrationService service = new RegistrationService(monitor, participantStore, dtoTransformerRegistry, new Telemetry());
```

Comment:
Can we use mock same as others e.g. Telemetry telemetry = mock(Telemetry.class);

Reply:
We thought about it but the constructor here is used on purpose as it returns OpenTelemetry.noop() which should be used for testing.

Reply:
after double checking the tests pass so I changed to mock(Telemetry.class) as suggested to be consistent